packages/block-library/src/post-author-name/index.php

```diff
@@ -27,10 +27,10 @@ function render_block_core_post_author_name( $attributes, $content, $block ) {
 	$align_class_name = empty( $attributes['textAlign'] ) ? '' : "has-text-align-{$attributes['textAlign']}";
 
 	if ( isset( $attributes['isLink'] ) && $attributes['isLink'] ) {
-		$author_name = sprintf( '<a href="%1$s" target="%2$s" class="wp-block-author-name__link">%3$s</a>', get_author_posts_url( $author_id ), $attributes['linkTarget'], $author_name );
+		$author_name = sprintf( '<a href="%1$s" target="%2$s" class="wp-block-author-name__link">%3$s</a>', get_author_posts_url( $author_id ), esc_attr( $attributes['linkTarget'] ), $author_name );
 	}
 
-	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $align_class_name ) );
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => esc_attr( $align_class_name ) ) );
```

Comment:
The `get_block_wrapper_attributes` function already runs `esc_attr` on all attribute values. Adding it here would cause double-escaping.

Reply:
I agree here that this isn't necessary.